### PR TITLE
feat: add fonts, and correct styles for home page

### PR DESCRIPTION
**Pull request by artur-lapaev**

### CODE REVIEW

File: src/app/components/app-list/app-list.component.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   display: flex;
   flex-direction: column;
   overflow: hidden;
-  font-family: Roboto Regular;
+  // font-family: Roboto Regular;
```
Comment (vharadkou) [inline src/app/components/app-list/app-list.component.scss]: please remove unused commented lines

File: src/assets/Fonts/Didact_Gothic/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2009, 2010, 2011 Daniel Johnson (<il.basso.buffo@gmail.com>).
```
Comment (vharadkou) [inline src/assets/Fonts/Didact_Gothic/OFL.txt]: 'Fonts' folder should start from lowercase

File: src/app/components/app-list/app-list.component.scss (hunk 1 above)
Comment (artur-lapaev) [reply src/app/components/app-list/app-list.component.scss]: i removed unused lines

File: src/assets/Fonts/Didact_Gothic/OFL.txt (hunk 2 above)
Comment (artur-lapaev) [reply src/assets/Fonts/Didact_Gothic/OFL.txt]: I renamed this folder.

**Pull request merged**